tests/unit/test_transformations.py

```diff
@@ -158,21 +158,21 @@ def test_composed_transformation(self, parameters):
             )
 
     @pytest.mark.unit
-    def test_verify_input(self, parameters):
+    def testverify_input(self, parameters):
```

Comment:
```suggestion
    def test_verify_input(self, parameters):
```

perhaps the underscore was removed in error?